test/Microsoft.ML.TestFramework/TestCommandMore.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Data;
+using System;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Runtime.RunTests
+{
+    public sealed partial class TestCommandMore : TestSteppedDmCommandBase
+    {
+        public TestCommandMore(ITestOutputHelper helper) : base(helper)
+        {
+        }
+
+        [Fact]
```

Comment:
I don't think we should be adding more tests to the `TestFramework` library.  I think the `TestFramework` library should just have common code, and the tests should be in other libraries.

Reply:
what should be the correct place for these tests ?
```CommandCVLinearClassification ``` could be moved to predictors.cs
but what about 
```
CommandTrainScorePriorTrainer
CommandVersionTypeCheck
```

Reply:
@TomFinley - thoughts?